src/Dapr.Workflow/WorkflowServiceCollectionExtensions.cs

```diff
@@ -57,7 +59,18 @@ public static class WorkflowServiceCollectionExtensions
 
                 if (TryGetGrpcAddress(out string address))
                 {
-                    builder.UseGrpc(address);
+                    var apiToken = Environment.GetEnvironmentVariable("DAPR_API_TOKEN");
```

Comment:
there is this function https://github.com/dapr/dotnet-sdk/blob/master/src/Shared/DaprDefaults.cs#L29 which you can use to read the DAPR_API_TOKEN env var and avoid duplicating its name multiple times DAPR_API_TOKEN